patch-package
--- a/node_modules/@tensorflow-models/posenet/dist/checkpoint_loader.js
+++ b/node_modules/@tensorflow-models/posenet/dist/checkpoint_loader.js
@@ -1,6 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-var tfjs_1 = require("@tensorflow/tfjs");
+var tfjs_1 = require("@tensorflow/tfjs-node");
 var MANIFEST_FILE = 'manifest.json';
 var CheckpointLoader = (function () {
     function CheckpointLoader(urlPath) {
--- a/node_modules/@tensorflow-models/posenet/dist/checkpoints.js
+++ b/node_modules/@tensorflow-models/posenet/dist/checkpoints.js
@@ -4,19 +4,19 @@ var mobilenet_1 = require("./mobilenet");
 var BASE_URL = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
 exports.checkpoints = {
     1.01: {
-        url: BASE_URL + 'mobilenet_v1_101/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_101/',
         architecture: mobilenet_1.mobileNetArchitectures[100]
     },
     1.0: {
-        url: BASE_URL + 'mobilenet_v1_100/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_100/',
         architecture: mobilenet_1.mobileNetArchitectures[100]
     },
     0.75: {
-        url: BASE_URL + 'mobilenet_v1_075/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_075/',
         architecture: mobilenet_1.mobileNetArchitectures[75]
     },
     0.5: {
-        url: BASE_URL + 'mobilenet_v1_050/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_050/',
         architecture: mobilenet_1.mobileNetArchitectures[50]
     }
 };
--- a/node_modules/@tensorflow-models/posenet/dist/mobilenet.js
+++ b/node_modules/@tensorflow-models/posenet/dist/mobilenet.js
@@ -1,5 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
+// TODO : change this when tfjs-node is compatible
 var tf = require("@tensorflow/tfjs");
 var mobileNet100Architecture = [
     ['conv2d', 2],
--- a/node_modules/@tensorflow-models/posenet/dist/posenet.js
+++ b/node_modules/@tensorflow-models/posenet/dist/posenet.js
@@ -1,7 +1,7 @@
 // @tensorflow/tfjs-models Copyright 2019 Google
 (function (global, factory) {
-  typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@tensorflow/tfjs')) :
-  typeof define === 'function' && define.amd ? define(['exports', '@tensorflow/tfjs'], factory) :
+  typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@tensorflow/tfjs-node')) :
+  typeof define === 'function' && define.amd ? define(['exports', '@tensorflow/tfjs-node'], factory) :
   (factory((global.posenet = {}),global.tf));
 }(this, (function (exports,tf) { 'use strict';
 
--- a/node_modules/@tensorflow-models/posenet/dist/posenet_model.js
+++ b/node_modules/@tensorflow-models/posenet/dist/posenet_model.js
@@ -36,7 +36,7 @@ var __generator = (this && this.__generator) || function (thisArg, body) {
 };
 var _this = this;
 Object.defineProperty(exports, "__esModule", { value: true });
-var tf = require("@tensorflow/tfjs");
+var tf = require("@tensorflow/tfjs-node");
 var checkpoint_loader_1 = require("./checkpoint_loader");
 var checkpoints_1 = require("./checkpoints");
 var mobilenet_1 = require("./mobilenet");
--- a/node_modules/@tensorflow-models/posenet/dist/single_pose/argmax2d.js
+++ b/node_modules/@tensorflow-models/posenet/dist/single_pose/argmax2d.js
@@ -1,6 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-var tf = require("@tensorflow/tfjs");
+var tf = require("@tensorflow/tfjs-node");
 function mod(a, b) {
     return tf.tidy(function () {
         var floored = a.div(tf.scalar(b, 'int32'));
--- a/node_modules/@tensorflow-models/posenet/dist/single_pose/util.js
+++ b/node_modules/@tensorflow-models/posenet/dist/single_pose/util.js
@@ -1,6 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-var tf = require("@tensorflow/tfjs");
+var tf = require("@tensorflow/tfjs-node");
 var keypoints_1 = require("../keypoints");
 function getPointsConfidence(heatmapScores, heatMapCoords) {
     var numKeypoints = heatMapCoords.shape[0];
--- a/node_modules/@tensorflow-models/posenet/dist/util.js
+++ b/node_modules/@tensorflow-models/posenet/dist/util.js
@@ -35,7 +35,7 @@ var __generator = (this && this.__generator) || function (thisArg, body) {
     }
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-var tf = require("@tensorflow/tfjs");
+var tf = require("@tensorflow/tfjs-node");
 var keypoints_1 = require("./keypoints");
 function eitherPointDoesntMeetConfidence(a, b, minConfidence) {
     return (a < minConfidence || b < minConfidence);
