patch-package
--- a/node_modules/@tensorflow-models/posenet/dist/checkpoints.js
+++ b/node_modules/@tensorflow-models/posenet/dist/checkpoints.js
@@ -4,19 +4,19 @@ var mobilenet_1 = require("./mobilenet");
 var BASE_URL = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
 exports.checkpoints = {
     1.01: {
-        url: BASE_URL + 'mobilenet_v1_101/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_101/',
         architecture: mobilenet_1.mobileNetArchitectures[100]
     },
     1.0: {
-        url: BASE_URL + 'mobilenet_v1_100/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_100/',
         architecture: mobilenet_1.mobileNetArchitectures[100]
     },
     0.75: {
-        url: BASE_URL + 'mobilenet_v1_075/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_075/',
         architecture: mobilenet_1.mobileNetArchitectures[75]
     },
     0.5: {
-        url: BASE_URL + 'mobilenet_v1_050/',
+        url: (POSENET_BASE_URL || BASE_URL) + 'mobilenet_v1_050/',
         architecture: mobilenet_1.mobileNetArchitectures[50]
     }
 };
